ui/app/src/components/RejectDialog/RejectDialogUI.tsx

```diff
@@ -123,7 +135,7 @@ export function RejectDialogUI(props: RejectDialogUIProps) {
           <PrimaryButton
             onClick={onReject}
             loading={isSubmitting}
-            disabled={checkedItems.length === 0 || rejectionComment === '' || rejectionReason === '' || isSubmitting}
+            disabled={checkedItems.length === 0 || rejectionComment === '' || isSubmitting}
```

Comment:
Let's move this logic up. Add a `isSubmitDisabled` prop that you use here and pass it down from the container.